core/src/main/java/com/arcrobotics/ftclib/hardware/motors/MotorGroup.java

```diff
@@ -1,14 +1,21 @@
 package com.arcrobotics.ftclib.hardware.motors;
 
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
 /**
  * Allows multiple {@link Motor} objects to be linked together
  * as a single group. Multiple motors will act together.
  *
  * @author Jackson
  */
-public class MotorGroup extends Motor {
+public class MotorGroup extends Motor implements Iterable<Motor> {
 
-    private Motor[] group;
+    private final Motor[] group;
```

Comment:
This is a breaking change, the PR description should be changed.

Reply:
I don't believe this breaks anything on the user end?

Reply:
If you inherited from `MotorGroup`, and overrode `group`, your code will not compile with the new version. Granted, doing that is... weird but it's possible and therefore it's a breaking change.